tests/03_generic_output/09_observe_output.yaml

```diff
@@ -57,6 +57,8 @@ problem: !Coupling_Sequential
       end_time: 1000000.0
     balance:
       cumulative: true
+      times:
+        - 20000.0
```

Comment:
We should not search for inputs that leads to reference outputs.
In particular for this test, removing time 20000 from mass_balance output is fine. 
More over I wonder why there are any times in the water_balance ??